tests/test_scipinstancecopy.py

```diff
@@ -0,0 +1,22 @@
+from pyscipopt import Model
+
+def test_scipinstancecopy():
+    # create solver instance
+    s = Model()
+
+    # add some variables
+    x = s.addVar("x", vtype = 'C', obj = 1.0)
+    y = s.addVar("y", vtype = 'C', obj = 2.0)
+    s.setObjective(4.0 * y, clear = False)
+
+    c = s.addCons(x + 2 * y >= 1.0)
+
+    # solve problems
+    s.optimize()
+
+    s2 = s.createModelFromSCIP()
```

Comment:
This does not look very "Pythonic". Can't we call it `s.copy()`?

Reply:
It is not very "Pythonic", it is more in the style of the PySCIPOpt functions, such as `includeDefaultPlugins` and `createProbBasic`.

`s.copy()` would be confusing because this function doesn't actually copy SCIP, but copies the python `Model`. What do you think about `s.copyModel()` or `s.copy_model()`?

Reply:
`crateProbBasic` and `includeDefaultPlugins` are simple oneline wrappers and are named after the SCIP functions. My point is that in the best case the user should not even have to know what a `SCIP*` is, let alone work with it. Does it make sense to not return a `SCIP*` at all in the benders code and instead create a Model around it right away?

Reply:
> Does it make sense to not return a SCIP* at all in the benders code and instead create a Model around it right away?
This is exactly what I am trying to do. A SCIP* is returned and then I want to access the internal data using the Model class.